src/bid/encoding.rs

```diff
@@ -9,58 +9,66 @@
 //! See: https://hackmd.io/@7dpNYqjKQGeYC7wMlPxHtQ/BkfS78Y9L
 
 use super::Bid;
+use dusk_bls12_381::BlsScalar;
+#[cfg(feature = "std")]
 use dusk_plonk::constraint_system::ecc::Point as PlonkPoint;
+#[cfg(feature = "std")]
 use dusk_plonk::prelude::*;
-use poseidon252::{sponge::sponge::*, StorageScalar};
+use poseidon252::sponge::sponge::*;
```

Comment:
`poseidon252::sponge::sponge::sponge_hash` is deprecated, as discussed. `poseidon252::sponge::hash` should be used instead

Reply:
I wanted to use the gadget. And there's no shorcut to the path needed. Anyway I refactored the imports in other places.